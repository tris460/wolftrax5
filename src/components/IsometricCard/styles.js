import styled from 'styled-components';

export const Card = styled.div`
  position: relative;
  width: 200px;
  height: 200px;
  transition: 0.5s;
  border: 1px solid var(--transparent-color);

  &:hover {
    transform: translateY(-40px);
  }
  &:before {
    content: '';
    position: absolute;
    top: -15px;
    left: 0;
    width: 101%;
    height: 15px;
    background: linear-gradient(
      to top,
      var(--transparent-color),
      var(--highligth)
    );
    transform-origin: bottom;
    transform: skewX(45deg);
    transition: 0.5s;
  }
  &:after {
    content: '';
    position: absolute;
    top: -15px;
    left: -15px;
    width: 15px;
    height: 100%;
    background: linear-gradient(
      to left,
      var(--transparent-color),
      var(--highligth)
    );
    transform-origin: left;
    transform: skewY(45deg);
    transition: 0.5s;
  }
`;

export const CardImgBox = styled.div`
  position: relative;
  width: 100%;
  height: 200px;
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;

  &:before {
    content: '';
    position: absolute;
    left: 0;
    bottom: 0;
    width: 100%;
    height: 200px;
    background: linear-gradient(
      transparent,
      transparent,
      var(--transparent-color)
    );
    transform-origin: bottom;
    transform: skewX(45deg);
    transition: 0.5s;
    pointer-events: none;
    z-index: -1;
    filter: blur(4px);
    opacity: 0.9px;
  }

  .card:hover &:before {
    transform: skewX(45deg) translateY(100px);
    filter: blur(10px);
    opacity: 0.5px;
  }
`;
