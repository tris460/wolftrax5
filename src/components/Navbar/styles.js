import styled from 'styled-components';
import { Link } from 'react-router-dom';

export const NavLink = styled(Link)`
  cursor: pointer;
  padding: 10px;
  margin: 5px;
  text-decoration: none;
`;
export const NavContainer = styled.nav`
  display: grid;
  height: 100%;
  grid-template-rows: 1fr 8fr 1fr;
  justify-items: center;
  align-items: center;

  @media (max-width: 950px) {
    grid-template-rows: 1fr;
    grid-template-columns: 1fr 8fr;
  }
`;
export const NavSection = styled.section`
  display: flex;
  flex-direction: column;
  justify-content: space-evenly;
  @media (max-width: 950px) {
    flex-direction: row;
  }
`;
