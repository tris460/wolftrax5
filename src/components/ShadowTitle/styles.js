import styled from 'styled-components';

export const H2 = styled.h2`
  color: #fff;
  display: flex;
  justify-content: center;
  align-items: center;
  font-size: 15em;
  text-shadow: -25px 25px 40px rgba(0, 0, 0, 0.5),
    -10px 10px 0 rgba(255, 255, 255, 0.5), -20px 20px 0 rgba(255, 255, 255, 0.2),
    -30px 30px 0 rgba(255, 255, 255, 0.05);
`;
