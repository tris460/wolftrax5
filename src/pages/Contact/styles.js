import styled from 'styled-components';

export const IsometricContainer = styled.div`
  width: 80%;
  margin-left: 10%;
  margin-top: 40px;
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-template-rows: repeat(3, 1fr);
  grid-row-gap: 75px;
  grid-column-gap: 20px;
  justify-items: space-between;

  transform: skewY(-10deg);

  @media (min-width: 1000px) {
    & a:nth-child(1) {
      grid-column: 1;
    }
    & a:nth-child(2) {
      grid-column: 3;
    }
    & a:nth-child(3) {
      grid-column: 2;
      grid-row: 2;
    }
    & a:nth-child(4) {
      grid-column: 1;
      grid-row: 3;
    }
    & a:nth-child(5) {
      grid-column: 4;
      grid-row: 3;
    }
  }

  @media (max-width: 950px) {
    grid-template-columns: 1fr;
    grid-template-rows: repeat(5, 1fr);
    grid-row-gap: 35px;
    font-size: 12px;
    justify-items: center;
    & a:nth-child() {
      grid-column: 1;
    }
  }
`;
