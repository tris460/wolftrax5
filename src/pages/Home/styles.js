import styled from 'styled-components';

export const MainContainer = styled.main`
  font-size: 18px;
  display: grid;
  width: 100%;
  height: 100%;
  grid-template-columns: 8fr 2fr;

  @media (max-width: 950px) {
    grid-template-rows: 8fr 2fr;
    grid-template-columns: 1fr;
    font-size: 12px;
  }
`;

export const HomeContainer = styled(MainContainer)``;

export const SectionInfo = styled.section`
  place-self: center;
  z-index: 100;
  & h1 {
    font-size: 78px;
  }
  @media (max-width: 950px) {
    & h1 {
      text-align: center;
      font-size: 58px;
    }
  }
`;
