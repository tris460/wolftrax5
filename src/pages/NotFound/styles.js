import styled from 'styled-components';

export const SectionNotFound = styled.section`
  display: flex;
  justify-content: center;
  align-items: center;
  min-height: 100vh;
  background: linear-gradient(
    135deg,
    var(--highligth),
    var(--highligth) 50%,
    var(--color-gray) 50%,
    var(--color-gray)
  );
`;
