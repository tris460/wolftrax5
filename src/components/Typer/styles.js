import styled from 'styled-components';
import { blink } from '../../styles/animations';

export const TextContainer = styled.span`
  color: ${(props) => props.color || 'var(--highligth)'};
  background-color: ${(props) => props.bg || 'trasparent'};
  font-size: ${(props) => props.size || 12}px;
  font-family: ${(props) => props.font};
  text-align: center;
  @media (max-width: 950px) {
    font-size: 12px;
  }
`;
export const Cursor = styled.span`
  ${blink({ time: '0.8s' })}
`;
