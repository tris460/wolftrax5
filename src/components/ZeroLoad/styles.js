import styled from 'styled-components';
import { rotate } from '../../styles/animations';

export const Zero = styled.span`
  position: relative;
  display: inline-block;
  width: 200px;
  height: 200px;
  background: linear-gradient(to bottom, #fff, #777);
  border-radius: 50%;
  margin: 0 15px;
  box-shadow: -25px 25px 40px rgba(0, 0, 0, 0.5);
  &::before {
    content: '';
    position: absolute;
    top: 50%;
    right: 0;
    transform: translateY(-50%);
    width: 180px;
    height: 180px;
    background: linear-gradient(to top, #fff, #777);
    border-radius: 50%;
  }
  &::after {
    content: '';
    position: absolute;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
    width: 100px;
    height: 100px;
    border-radius: 50%;
    background: linear-gradient(
      315deg,
      var(--highligth),
      var(--highligth) 49%,
      var(--color-gray) 51%,
      var(--color-gray)
    );
    box-shadow: inset 0 0 30px rgba(0, 0, 0, 1);
    ${rotate()}
  }
`;
