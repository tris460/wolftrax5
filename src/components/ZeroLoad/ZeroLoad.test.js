import React from 'react';
import { shallow } from 'enzyme';
import { create } from 'react-test-renderer';

import { ZeroLoad } from './index';
import { Zero } from './styles';

describe('<ZeroLoad />', () => {
  const zero = shallow(<ZeroLoad />);

  test('Render of Component ZeroLoad', () => {
    expect(zero.length).toEqual(1);
  });
});

describe('ZeroLoad Snapshot', () => {
  test('Check UI of component ZeroLoad', () => {
    const zero = create(<ZeroLoad />).toJSON();
    expect(zero).toMatchSnapshot();
  });
  test('Check UI of component Zero', () => {
    const zero = create(<Zero />).toJSON();
    expect(zero).toMatchSnapshot();
  });
});
