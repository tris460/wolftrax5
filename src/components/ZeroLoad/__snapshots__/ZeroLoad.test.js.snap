// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ZeroLoad Snapshot Check UI of component Zero 1`] = `
<span
  className="sc-bdfBwQ ecsZWe"
/>
`;

exports[`ZeroLoad Snapshot Check UI of component ZeroLoad 1`] = `
<span
  className="sc-bdfBwQ ecsZWe"
/>
`;
